File: c/FizzBuzz.c
#include <stdio.h>

int main(){
  for(int i = 1 ; i <= 100 ; i++){
    if(i % 3 == 0){
      if(i % 5 == 0){
	printf("FizzBuzz\n");
      }else{
	printf("Fizz\n");
      }
    }else if(i % 5 == 0){
      printf("Buzz\n");
    }else{
      printf("%d\n",i);
    }
  }
}

File: python/FizzBuzz.py
import sys

def fizzbuzz(x):
    if x % 3 == 0:
        if x % 5 == 0:
            return 'FizzBuzz'
        else:
            return 'Fizz'
    elif x % 5 == 0:
        return 'Buzz'
    else:
        return str(x)

print "\n".join(fizzbuzz(int) for int in range(1,101))
